doc/installation.rst

```diff
@@ -32,20 +32,23 @@ Just visit the `Spyder page on Binder`_ to get started using Spyder now.
 
 
 
-======================================
-Installing with Anaconda (recommended)
-======================================
+==============
+Using Anaconda
+==============
+
+.. important::
+
+   If in doubt, you should install Spyder via this method; it generally has the least likelihood of potential pitfalls for non-experts, and we may be able to provide limited assistance if you do run into trouble.
```

Comment:
```suggestion
   If in doubt, you should install Spyder via this method; it generally has the least likelihood of potential pitfalls for non-experts, and we may provide assistance if you do run into trouble.
```

Reply:
```suggestion
   If in doubt, you should install Spyder via this method; it generally has the least likelihood of potential pitfalls for non-experts, and we may be able to provide assistance if you do run into trouble.
```

Good point, but it still needs the "be able to" to sound right.

Reply:
Ok, could we rephrase it a bit then? We're giving a ton of assistance right now, but (as a non-native speaker), this sounds to me "we could help you", whereas I'd like to send a message closer to "if you install Anaconda, we'll definitely help you".

Reply:
Ah, I see. Its not just you, "we could help you" is the in fact the precise message it was originally intended to send; in my experience with user tech support I've found its almost always better to under- than over-promise, exceed people's expectations rather than risk disappointing them, and let our actions do the talking. But if we really want to promise that, what we could say is just

```suggestion
   If in doubt, you should install Spyder via this method; it generally has the least likelihood of potential pitfalls for non-experts, and we will be able to provide assistance if you do run into trouble.
```

Reply:
Great! I agree with this message now!

Thanks a lot @CAM-Gerlach for your patience with my requests.

Reply:
Same to you for your patience with me :smile: